src/app/postcode.service.spec.ts

```diff
@@ -1,18 +1,18 @@
 import { provideHttpClientTesting } from '@angular/common/http/testing';
 import { TestBed } from '@angular/core/testing';
 
-import { PostcodeService } from './postcode.service';
+import { AddressService } from './address.service';
 import { provideHttpClient, withInterceptorsFromDi } from '@angular/common/http';
 
 describe('PostcodeService', () => {
```

Comment:
Probably slightly better to rename the file and change its `describe(...)` to avoid confusion

Reply:
Yep, done in 23606d24